### DISCUSSION TITLE
Container App

### DISCUSSION THREAD

**Discussion by sandeep-eroad** [CompositionalIT/farmer#946] [Q&A]
We are evaluating Farmer to deploy Azure Container apps, I have following questions

1. Is it possible to setup Continuous deployment using Farmer?
2. Should we have one container app with different containers or multiple container apps so that Continuous deployments can work for multiple repos? We anticipate use of .net and java.

**Comment by isaacabraham** [comment]
@sandeep-eroad Hi!

1. Yes, you definitely can and we do. Farmer is just .NET code which can run from any CI/CD platform.
2. That depends on your use case :-) Bear in mind that Container Apps work with Docker; when you deploy new versions of code, you don't actually need to redeploy the container app itself, but simply update the container registry that the Container App is linked to - it should automatically pull through the newest version
